components/arm/trossen/trossen.go

```diff
@@ -416,6 +487,10 @@ func (a *Arm) SleepPosition(ctx context.Context) error {
 	a.JointTo("Wrist", 2509, sleepWait)
 	a.JointTo("Forearm_rot", 2048, sleepWait)
 	a.JointTo("Elbow", 3090, sleepWait)
+	if err := a.OpenGripper(ctx); err != nil {
+		a.moveLock.Unlock()
```

Comment:
Why is this here? OpenGripper does its own lock/unlock on this. If anything, it needed to be unlocked BEFORE this.